src/components/Button.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const Button = ({
+  children,
+  className,
+  onClick,
+  large,
+  clicked,
+  index,
+  isDisabled,
+}) => {
+  return (
+    <>
+      <button
+        className={` h-20 border-gray-500 border text-center w-full ${className} ${
+          large && "col-span-2"
+        } ${
+          clicked === index &&
+          clicked !== undefined &&
+          "bg-indigo-100 text-orange-800"
+        } `}
+        onClick={onClick}
+        disabled={isDisabled && children === "." ? true : false}
```

Comment:
You can change the ternary to a boolean expression which is more readable.
```suggestion
        disabled={isDisabled && children === "."}
```